codegen/src/test/resources/software/amazon/awssdk/codegen/poet/auth/scheme/query-endpoint-auth-params-auth-scheme-modeled-provider.java

```diff
@@ -27,13 +27,13 @@
 
 @Generated("software.amazon.awssdk:codegen")
 @SdkInternalApi
-public final class DefaultQueryAuthSchemeProvider implements QueryAuthSchemeProvider {
-    private static final DefaultQueryAuthSchemeProvider DEFAULT = new DefaultQueryAuthSchemeProvider();
+public final class InternalModeledQueryAuthSchemeProvider implements QueryAuthSchemeProvider {
+    private static final InternalModeledQueryAuthSchemeProvider DEFAULT = new InternalModeledQueryAuthSchemeProvider();
```

Comment:
nit: The design uses "Internal" but in Java SDK, given the impls are already internal, if we should drop "Internal" prefix. `Modeled{ServiceId}AuthSchemeProvider`?

Reply:
Yeah, makes sense, I will change in a follow up PR.